### ISSUE TITLE
module.OnApplicationShutdown can not execute.

### ISSUE THREAD

**Issue by jiyangcd** [abpframework/abp#635]
```
 public override void OnApplicationShutdown(ApplicationShutdownContext context)
        {
            var logger = context.ServiceProvider.GetRequiredService<ILogger<GApiModule>>();
            logger.LogDebug("GApiModule.OnApplicationShutdown ");
        }
```
i want do something wen application shutdown ,but OnApplicationShutdown not work in my system ,did I use the wrong way？

**Comment by hikalkan** [comment]
It's not being called yet. So, it does not work :)

See https://github.com/abpframework/abp/issues/265

**Comment by maliming** [comment]
fixed by #265
